src/lib/utils.js

```diff
@@ -71,7 +71,10 @@ _.each([
 
   _['isArrayOf' + type + 's'] = function (arr) {
     // quick shallow check of arrays
-    return _.isArray(arr) && _.every(arr.slice(0, 10), check);
+    return _.isArray(arr) && _.every(arr.slice(0, 10), function (item) {
+
+      return check(item);
```

Comment:
`_.every()` calls the function with `(item, index)` - if we directly pass that through - node 8 crashes, because `isRegexp` check expects a single argument; the fix is to make sure we call `check()` with a single argument.

I do wonder, though, if all of the `isArrayOf...` stuff should be removed altogether, as the only time I see this used is a single `isArrayOfStrings()` call.

Also curious about the `slice()` - is that for performance reasons that this only checks the first 10 members of an array?

Reply:
yeah, I was probably being overly ambitious when I first wrote these utils and thought I would use them for something... Feel free to delete anything that's not used and inline little helpers like `isArrayofString()` rather than keeping it in `lib/utils` if it's only used once.